.jpmignore

```diff
@@ -7,6 +7,7 @@ CONTRIBUTING.md
 *.zip
 *.xpi
 *.xcf
-node_modules
+node_modules/st
+node_modules/.bin
```

Comment:
> Why's this here?

Answering another of my own questions, looks like we want jpm to pick up some but not all packages. I looked at `.jpmignore` but saw `.gitignore`.